src/scvi/external/decipher/_model.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from collections.abc import Sequence
+
+import numpy as np
+import pyro
+import torch
+from anndata import AnnData
+
+from scvi._constants import REGISTRY_KEYS
+from scvi.data import AnnDataManager
+from scvi.data.fields import LayerField
+from scvi.model.base import BaseModelClass, PyroSviTrainMixin
+from scvi.train import PyroTrainingPlan
+from scvi.utils import setup_anndata_dsp
+
+from ._module import DecipherPyroModule
+from ._trainingplan import DecipherTrainingPlan
+
+logger = logging.getLogger(__name__)
+
+
+class Decipher(PyroSviTrainMixin, BaseModelClass):
+    """Decipher model for single-cell data analysis :cite:p:`Nazaret23`.
+
+    Parameters
+    ----------
+    adata
+        AnnData object that has been registered via
+        :meth:`~scvi.model.Decipher.setup_anndata`.
+    dim_v
+        Dimension of the interpretable latent space v.
```

Comment:
better word than interpretable?

Reply:
Unfortunately not, since it captures the same info as z, just meant to be more directly interpretable